doc/spec/bikeshed/slam.bs

```diff
@@ -189,7 +189,6 @@ interface Instance {
 :: *Return type*:Promise&lt;<a>String</a>&gt;
 
 : {{loadOccupancyMap}}
-:: Note: not supported yet.
```

Comment:
Another "not supported yet" at line-90.
https://github.com/lisa0314/node-realsense-1/blob/120f006ebf63c6a0b3fa0349748bf3444b60db55/doc/spec/bikeshed/slam.bs#L90